### PR TITLE
[FSDE-10859] Fix checkbox issue for multiple lines

### PR DESCRIPTION
**Pull request by soujanreddy**

### CODE REVIEW

**Comment by SCMAUTO** [comment]
The CI pipeline did not run successfully in https://jenkinsprod.corp.intranet:8443/job/UX-CHI/job/Productive/job/Chi/job/PR-248/1/. :x:

**Comment by SCMAUTO** [comment]
The CI pipeline did not run successfully in https://jenkinsprod.corp.intranet:8443/job/UX-CHI/job/Productive/job/Chi/job/PR-248/2/. :x:

**Comment by SCMAUTO** [comment]
The CI pipeline did not run successfully in https://jenkinsprod.corp.intranet:8443/job/UX-CHI/job/Productive/job/Chi/job/PR-248/3/. :x:

**Comment by SCMAUTO** [comment]
The CI pipeline run successfully in https://jenkinsprod.corp.intranet:8443/job/UX-CHI/job/Productive/job/Chi/job/PR-248/4/. :white_check_mark:

**Comment by SCMAUTO** [comment]
You can check this PRs instance in https://81e4f27053df225791c6e248d8a5289e47bd65bf-chi-pr.enterprise.ctl.io (external) and in https://nginx-pr-248-ux-chi.kubeodc-test.corp.intranet/ (internal)

**Comment by SCMAUTO** [comment]
The CI pipeline run successfully in https://jenkinsprod.corp.intranet:8443/job/UX-CHI/job/Productive/job/Chi/job/PR-248/5/. :white_check_mark:

**Comment by SCMAUTO** [comment]
You can check this PRs instance in https://7e8efa9ea9b472706b74484cc888bdd621b1d28f-chi-pr.enterprise.ctl.io (external) and in https://nginx-pr-248-ux-chi.kubeodc-test.corp.intranet/ (internal)

**Comment by SCMAUTO** [comment]
The CI pipeline did not run successfully in https://jenkinsprod.corp.intranet:8443/job/UX-CHI/job/Productive/job/Chi/job/PR-248/6/. :x:

**Comment by SCMAUTO** [comment]
The CI pipeline did not run successfully in https://jenkinsprod.corp.intranet:8443/job/UX-CHI/job/Productive/job/Chi/job/PR-248/7/. :x:

**Review by mattnickles — APPROVED** [review]

**Review by evansito — APPROVED** [review]

File: src/chi/components/checkbox/checkbox.scss [hunk 1]
```diff
@@ -33,9 +33,9 @@
         content: '';
         display: block;
         height: 1rem;
+        top: 0.1rem;
```
Comment (jllr) [inline src/chi/components/checkbox/checkbox.scss:36]: Please revert to the previous line and value

File: src/chi/components/checkbox/checkbox.scss [hunk 2]
```diff
@@ -45,9 +45,9 @@
         content: '';
         display: block;
         height: 0.625rem;
+        top: 0.3rem;
```
Comment (jllr) [inline src/chi/components/checkbox/checkbox.scss:48]: Please revert to the previous line and set value to 0.3125rem

**Review by jllr — CHANGES_REQUESTED** [review]

**Comment by SCMAUTO** [comment]
The CI pipeline run successfully in https://jenkinsprod.corp.intranet:8443/job/UX-CHI/job/Productive/job/Chi/job/PR-248/8/. :white_check_mark:

**Comment by SCMAUTO** [comment]
You can check this PRs instance in https://bb1ec1031c96154b443af6ea8318c4bf11609293-chi-pr.enterprise.ctl.io (external) and in https://nginx-pr-248-ux-chi.kubeodc-test.corp.intranet/ (internal)

**Review by janibolkvadze — APPROVED** [review]

**Comment by SCMAUTO** [comment]
The CI pipeline run successfully in https://jenkinsprod.corp.intranet:8443/job/UX-CHI/job/Productive/job/Chi/job/PR-248/9/. :white_check_mark:

**Comment by SCMAUTO** [comment]
You can check this PRs instance in https://b6f38959b4e5ca2e0b80e4ce97908922fe1c83a9-chi-pr.enterprise.ctl.io (external) and in https://nginx-pr-248-ux-chi.kubeodc-test.corp.intranet/ (internal)

**Comment by SCMAUTO** [comment]
The CI pipeline run successfully in https://jenkinsprod.corp.intranet:8443/job/UX-CHI/job/Productive/job/Chi/job/PR-248/10/. :white_check_mark:

**Comment by SCMAUTO** [comment]
You can check this PRs instance in https://eac602084b26c6cf09ce516bc1798bc2bce60a95-chi-pr.enterprise.ctl.io (external) and in https://nginx-pr-248-ux-chi.kubeodc-test.corp.intranet/ (internal)

**Review by jllr — APPROVED** [review]

**Pull request merged**